_rules/95e043.md

```diff
@@ -0,0 +1,90 @@
+---
+name: Auto-play audio has a control mechanism
+test_type: atomic
+
+description: |
+  This rule checks that each auto-play audio or video element has a control mechanism.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `audio` or `video` element with a `src` file content `duation` of more than 3 seconds, is set to `autoplay` and or `loop`, and has audio output that is not `paused` or `muted`.
+
+## Expectation 1
+
+For each test target a control mechanism is provided to pause or stop the audio, or turn the audio volume off independently from the overall system volume control.
+
+## Expectation 2
+
+The control mechanism to pause or stop or turn the audio volume off, is [visible](#visible), has an [accessible name](#accessible-name) and is one of the first five focusable elements in the [document](#https://www.w3.org/TR/dom/#concept-document).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
+## Test Cases
+
+### Passed
+
+### Passed example 1
+
+The `audio` element has a control mechanism.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" autoplay="true" controls></audio>
+```
+
+### Failed
+
+### Failed example 1
+
+The `audio` element autoplays and does not have a control mechanism.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" autoplay="true" controls></audio>
+```
+
+### Inapplicable
+
+### Inapplicable example 1
+
+The `video` element is `muted`.
+
+``` html
+ <video autoplay="true" muted="true">
+  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+### Inapplicable example 2
+
+The `video` element has no audio output.
```

Comment:
Is it that the video has no audio, or that the element has no audio because it is muted? If it is the latter then this example is the same as the previous.

Reply:
The `scr` file doesn't output any audio as opposed it audio being muted - that was the intention. I have updated this accordingly.